astropy/io/fits/hdu/hdulist.py

```diff
@@ -510,6 +510,46 @@ def fileinfo(self, index):
 
         return output
 
+    def __copy__(self):
+        """
+        Return a shallow copy of the object.
+
+        Returns
+        -------
+        self : HDUList
+            A shallow copy of this :class:`HDUList` object.
+
+        """
+
+        return self[:]
+
+    def copy(self):
```

Comment:
You could even use `copy = __copy__` and avoid the layer of indirection.

Reply:
Wow! Thanks. I had no idea it was possible to something like this for a class method.

Reply:
Wow! Thanks. I had no idea it was possible to do something like this for a class method.